### DISCUSSION TITLE
State of roadmap 2023

### DISCUSSION THREAD

**Discussion by luke-z** [primefaces/community#688] [PrimeVue] [answered]
Hello everyone

I'm wondering if the roadmap for Q4'23 is still on track.

I'm really excited for "CSS Variables" and "New UI Based Theme Editor". Do you think this will be available soon or rather Q1/2 in '24?

Custom theming has been a pain lately, as the designer is out-dated and manually adjusting the css/scss variables is not really nice to work with 😊

Best regards

**Comment by mertsincan** [comment, marked as answer]
Hello,

Glad to hear that you're excited! We started working on CSS variables in PrimeVue. Q1 will also be completed. Then, other libraries start porting with a similar structure. We plan to implement the New theme editor in Q2/3. We (especially me)  focused entirely on these two features.

https://www.reddit.com/r/vuejs/comments/18goqso/introducing_the_allnew_tailwind_css_based_ui/

Best Regards,
